lambda-utils/lib/lambda-utils.ts

```diff
@@ -8,6 +13,28 @@ const logger = new Logger('lambda-utils');
 /** Define the async version of ProxyHandler */
 export type AsyncProxyHandler = (event: APIGatewayEvent, context: Context) => Promise<any>;
 
+/**
+ * Casted interface for APIGatewayProxyEvents converted through the middleware
+ *
+ * @typedef {Interface}
+ */
+export interface APIGatewayProxyEvent extends AWS_APIGatewayProxyEvent {
+    /**
+     * HTTP Request body, parsed from a JSON string into an object.
+     */
+    body: any | null;
+
+    /**
+     * HTTP Path Parameters, parsed from a JSON string into an object
+     */
+    pathParameters: { [name: string]: string };
+
+    /**
+     * HTTP URL query string parameters, parsed from a JSON string into an object
```

Comment:
Sorry... just a pair of mistakes here in the descriptions of both of these two new properties. They are not parsed from JSON strings. What LambdaUtils does is guarantee that they are defined. So they will be an empty object rather than null if there are no path parameters or query string. Maybe the part after the comma on both of these would read "always defined, never null."

Reply:
Sorry just seeing this, I'll go in and make that change quick.

Reply:
https://github.com/onicagroup/sailplane/pull/8/commits/333404dc88be26d2a08299fa2430af5e46e0757a